### PR TITLE
Add Hostek.com

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/h/hostek.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Hostek.com": {
```
Comment (ApCoder123) [inline entries/h/hostek.com.json]:
```suggestion
  "Hostek": {
```

File: entries/h/hostek.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "domain": "hostek.com",
```
Comment (ApCoder123) [inline entries/h/hostek.com.json:3]:
```suggestion
    "domain": "hostek.com",
    "additional-domains": [
      "hostek.co.uk"
    ],
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Comment by 40504** [comment]
image

**Comment by 40504** [comment]
Also in terms & conditions, they called their self HOSTEK.COM

**Comment by 40504** [comment]
https://hostek.com/tos.html

**Comment by 40504** [comment]
@Carlgo11 @ApCoder123 what do you think about this?

**Comment by ApCoder123** [comment]
Hmm... They don't seem to be very consistent with their brand name. E.g. 

The `<title>` tag on their website uses "Hostek" rather than "Hostek.com"
image

and they refer to themselves as "Hostek" on Linkedin as well
image

Most of the entries on 2fa.directory don't tend to include a full domain name either, so I think it is probably fine how it is at the moment.

**Pull request merged**